.env_EXAMPLE

```diff
@@ -11,3 +11,4 @@ FLASKROOT="http://167.172.233.110"
 PORT="5000"
 HOST="http://127.0.0.1:5000"
 CUBECOBRA_VERSION="3.4.1"
+USE_S3="true"
```

Comment:
Default should be false since there are no aws credentials in the default env file.